src/scales/scale.linear.js

```diff
@@ -106,21 +113,28 @@ module.exports = function(Chart) {
 					var meta = chart.getDatasetMeta(datasetIndex);
 					if (chart.isDatasetVisible(datasetIndex) && IDMatches(meta)) {
 						helpers.each(dataset.data, function(rawValue, index) {
-							var value = +me.getRightValue(rawValue);
-							if (isNaN(value) || meta.data[index].hidden) {
+							var value = me._parseValue(rawValue);
+
+							if (isNaN(value.min) || isNaN(value.max) || meta.data[index].hidden) {
 								return;
 							}
 
+							if (value.min === 0) {
```

Comment:
hi @etimberg! We had to force to this check in order to keep the logic of scale building, its ok that 0 will not pass to scale values. Without this check some unit tests were not passed cause generally scale should not take account of 0 as start point. [0, 10] is not actually a float-bar, it has standartized representation as array but in fact its another view of 10. So, if we pass 10 this will be treated as [0,10]. So, scale should only take 10 and not zero. The reason why we should omit this 0 - for example we will use for bulding bars two values - 1 and 2. The new logic will treat them as [0,1] and [0,2]. WE have three uniq values here - 0, 1, 2. If scale will take all of them it will build scale from 0 to 2. According to general logic (without treating as arrays) it will build scale only from 1 to 2. We will have pixels difference in case of standart as integers or new logic (as arrays) values for graphs. SO this check is checking if the zero is actually a new value that should be passed to scale or its just for compability of old and new methods.